### ISSUE TITLE
Don’t ask for permission when checking updates

### ISSUE THREAD

**Issue by verbosus** [robofont-mechanic/mechanic-2#19]
…just check for updates and pop up the full Mechanic window if there are any available, especially since we already have a “Check for updates on startup” setting available and on by default, which some users might want to turn off.

Some preliminary discussion [here](https://github.com/robofont-mechanic/mechanic-2/pull/18#issuecomment-537419184) by @typemytype:

> I would keep the same pace: once a day to check for updates (as github does not allows super many requests) based on the mechanic pref.
> 
> and popup Mechanic when all extensions are checked, if this is possible as you dont know when a all requests are done. If not possible it could popup Mechanic when the first needs-update-extension is found.

**Comment by nicksherman** [comment]
Yes, this please! I actually thought the sporadic prompts were a bug and asked about it on the Discord:
https://discord.com/channels/1052516637489766411/1055056130222604309/1146815727098155008
